runc.go

```diff
@@ -61,11 +62,18 @@ const (
 type Runc struct {
 	// Command overrides the name of the runc binary. If empty, DefaultCommand
 	// is used.
-	Command      string
-	Root         string
-	Debug        bool
-	Log          string
-	LogFormat    Format
+	Command   string
+	Root      string
+	Debug     bool
+	Log       string
+	LogFormat Format
+	// Pdeathsig sets a signal the child process will receive when the
+	// parent dies.
+	//
+	// When Pdeathsig is set, command invocations will call os.LockOSThread
+	// to prevent OS thread termination from spuriously triggering the
+	// signal. See https://github.com/golang/go/issues/27505 and
```

Comment:
It would be worthwhile to explicitly say that a program with GOMAXPROCS=1 might hang in this case; callers should ensure they retain at least one unlocked thread.

Also a nit: it's runtime.LockOSThread not os.LockOSThread

Reply:
Added a note and fixed the reference to `LockOSThread`.